src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -184,6 +185,11 @@ public void DeletePeriodicBackupConfiguration(long backupTaskId)
             {
                 if (periodicBackup.TaskId == backupTaskId)
                 {
+                    if (periodicBackup.Name != null && 
+                        periodicBackup.Name.Equals(ServerWideBackupConfiguration.ConfigurationName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Why do we have this? This forces us to have just a single one. But do we have to do it this way?

Reply:
Do we need/want more than one?

Reply:
12 hours full / 15 minutes inc for S3.
Weekly to glacier

AS a good example, we need two of them for that.

Reply:
done